src/languages/brainfuck.js

```diff
@@ -35,6 +35,7 @@ export default function(hljs) {
       },
       {
         // this mode works as the only relevance counter
+        className: 'literal',
         begin: /(?:\+\+|--)/,
         contains: [LITERAL]
```

Comment:
I think it's supposed to be a look-ahead... does this also work?
```suggestion
        begin: /(?=\+\+|--)/,
        contains: [LITERAL]
```

Reply:
Yes, that also works.